plugins/inputs/sqlserver/sqlserver.go

```diff
@@ -2,6 +2,8 @@ package sqlserver
 
```

Comment:
All the SQL Related queries have been moved in their own file, what about putting the "standard" edition SQL Queries in their own file too?
I think this will be even cleaner as we won't have queries in the main GO file

Reply:
Yeah I was thinking about doing that ran out of time, will do that in next commit, will split V2 in own file and clean up queries to just have SQLServer queries in own file .

Reply:
Split V2 in latest commit - Will do SQLServer as separate file later.